### PR TITLE
GHC-132: Add image upload to the end of the dashboard

### PR DESCRIPTION
**Pull request by basrem**

### CODE REVIEW

File: configuration/openmrs/apps/clinical/dashboard.json [hunk 1]
```diff
@@ -317,9 +317,27 @@
         "type": "custom",
         "displayOrder": 16,
         "config": {
-          "title": "DASHBOARD_TITLE_APPOINTMENTS_KEY",
+          "title": " DASHBOARD_TITLE_APPOINTMENTS_KEY",
```
Comment (reagan-meant) [inline configuration/openmrs/apps/clinical/dashboard.json]: Remove the space added here

File: configuration/openmrs/apps/clinical/dashboard.json [hunk 2]
```diff
@@ -317,9 +317,27 @@
...
+        "isObservation": true,
+        "displayOrder": 17,
+        "dashboardConfig": {
+          "title": "DASHBOARD_TITLE_VACCINATION_DOCUMENT_KEY",
+          "showDetailsButton": true,
+          "conceptNames": ["Vaccination Document"],
+          "numberOfVisits": 3
+        },
+        "expandedViewConfig": {
+          "conceptNames": ["Image"],
```
Comment (reagan-meant) [inline configuration/openmrs/apps/clinical/dashboard.json]: Use the correct concept name

File: configuration/openmrs/i18n/clinical/locale_es.json [hunk 3]
```diff
@@ -123,7 +123,6 @@
   "DASHBOARD_TITLE_SOCIAL_ECONOMIC_KEY": "Informe Socio Economico",
   "DASHBOARD_TITLE_CHECK_OUT_KEY": "Check out",
   "DASHBOARD_TITLE_BMI": "IMC",
-  "DASHBOARD_TITLE_WEIGHT":"Peso"
-
-
+  "DASHBOARD_TITLE_WEIGHT":"Peso",
+  "DASHBOARD_TITLE_VACCINATION_DOCUMENT_KEY":"documento de vacunación"
```
Comment (reagan-meant) [inline configuration/openmrs/i18n/clinical/locale_es.json]: Change to "Documento de vacunación"

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/openmrs/apps/clinical/dashboard.json (hunk 1 above)
Comment (basrem) [reply configuration/openmrs/apps/clinical/dashboard.json]: Removed

File: configuration/openmrs/apps/clinical/dashboard.json (hunk 2 above)
Comment (basrem) [reply configuration/openmrs/apps/clinical/dashboard.json]: Corrected

File: configuration/openmrs/i18n/clinical/locale_es.json (hunk 3 above)
Comment (basrem) [reply configuration/openmrs/i18n/clinical/locale_es.json]: Changed

File: configuration/openmrs/apps/clinical/dashboard.json (hunk 2 above)
Comment (basrem) [reply configuration/openmrs/apps/clinical/dashboard.json]: Done

File: configuration/openmrs/apps/clinical/dashboard.json (hunk 1 above)
Comment (basrem) [reply configuration/openmrs/apps/clinical/dashboard.json]: Done

**Pull request merged**